.vsts-ci/linux.yml

```diff
@@ -95,24 +100,69 @@ stages:
     parameters:
       pool: ubuntu-20.04
 
+- stage: TestContainer
+  displayName: Test in a container
+  dependsOn: [BuildLinuxStage]
+  jobs:
+  - job: getContainerJob
+    displayName: Choose a container
+    pool:
+      vmImage: ubuntu-20.04
+    steps:
+    - checkout: self
+      clean: true
+
+    - checkout: Docker
+      clean: true
+
+    - pwsh: |
+        $matrix = ./PowerShell-Docker/build.ps1 -GenerateMatrixJson -FullJson -Channel preview | ConvertFrom-Json
+        $linuxImages = $matrix.preview |
+          Where-Object {$_.osversion -notlike '*windows*' -and $_.osversion -notlike '*nano*' -and $_.osversion -notlike 'alpine*' -and $_.osversion -notlike '*arm*'} |
```

Comment:
could this be `Where-Object -match "ubuntu|centos|mariner|` (etc)?

Reply:
or notmatch?

Reply:
```suggestion
          Where-Object {$_.osversion -notmatch 'windows|nano|alpine|arm'} |
```